machine/reference/scp.md

```diff
@@ -27,5 +29,50 @@ A file created remotely!
 Just like how `scp` has a `-r` flag for copying files recursively,
 `docker-machine` has a `-r` flag for this feature.
 
-In the case of transferring files from machine to machine, they go through the
-local host's filesystem first (using `scp`'s `-3` flag).
\ No newline at end of file
+In the case of transferring files from machine to machine,
+they go through the local host's filesystem first (using `scp`'s `-3` flag).
+
+## Understanding file paths for remote deployments
+
+If you copy files to a remote server with `docker-machine scp`
+in order to deploy an application (e.g., in the cloud), paths for
+files on the remote server should map to your local test environment.
+When you copy files from a local to remote system, put them in a
+path where `docker-compose` knows how to find them,
+to mount into a new container. This assumes that you will use the same model
+remotely as you do locally.
+
+For example, given this configuration for a `web` container in a
+[Compose file](/compose/compose-file/index.md), `docker-compose.yml`:
+
+```none
+web:
+  container_name: web
+  restart: always
+  build: ./web
+  expose:
+    - "8000"
+  links:
+    - postgres:postgres
+  volumes:
+    - /usr/src/app/static
+    - ./data:/usr/src/app/data
+  env_file: .env
+  command: /usr/local/bin/gunicorn --workers 4 --timeout 120 --bind :8000 app:app
+```
+
+* The first path given for
+[volumes](/compose/compose-file/index.md#volumes-volumedriver),
+`/usr/src/app/data`, is the path _inside_ the container.
+
+* The second path, `./data:/usr/src/app/data`, is the path where
```

Comment:
I'm a little confused what the example would like to illuminate -- you could certainly `docker-machine scp` a directory over and bind mount it in to a container, but if that were the case I'd:

1. Leave out a reference to the "in-container" volume (`/usr/src/app/data`) since it's not really relevant to the issue at hand
2. Illustrate with a simple example including `docker-machine scp`

e.g.,

For instance, imagine if you wanted to transfer your local directory `/Users/londoncalling/webapp` to a remote VM and bind mount it into a container on the remote host (we'll suppose the remote user is `ubuntu`). You could do something like this:

```
$ docker-machine scp -r /Users/londoncalling/webapp machinename:/home/ubuntu/webapp
```

Then write a `docker-compose` file which bind mounts it in:

```
version: "3.1"
services:
  webapp:
    image: alpine
    command: cat /app/root.php
    volumes:
    - "/home/ubuntu/webapp:/app"
```

And we can try it out like so:

```
$ eval $(docker-machine env machinename)
$ docker-compose run webapp
```